src/ui/reducers/timeline.ts

```diff
@@ -10,6 +10,15 @@ import { mergeSortedPointLists } from "ui/utils/timeline";
 
 const initialFocusWindow = getPausePointParams().focusWindow;
 
+// The backend has a hardcoded limit it will only load recordings
+// up to 3 minutes long when running routines, otherwise it bails out.
+// If the recording is longer than that, we _won't_ have data for
+// React, Redux, or Jump to Code.
+// This corresponds to `RoutineMaxWindowDurationSeconds` in the backend.
+// The client uses recording duration in milliseconds.
+// Use this to show a warning in the React and Redux panels.
+export const MAX_RECORDING_LENGTH_FOR_ROUTINES_MS = 3 * 60 * 1000;
```

Comment:
If we have to hard-code this on the client, it seems like the sort of thing that might make more sense to put in our capabilities code.